<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="test3"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.667">
  <properties />
            <testcase name="getAddressList_test" time="0.667" classname="com.edu.scene.test.AddressList_Test">
                <failure type="java.lang.IllegalStateException"
                  message="Connection pool shut down"
              >
<![CDATA[java.lang.IllegalStateException: Connection pool shut down
  org.apache.http.util.Asserts.check(Asserts.java:34)
  org.apache.http.pool.AbstractConnPool.lease(AbstractConnPool.java:191)
  org.apache.http.impl.conn.PoolingHttpClientConnectionManager.requestConnection(PoolingHttpClientConnectionManager.java:267)
  org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:176)
  org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
  org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
  org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
  org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
  org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
  org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
  com.edu.core.HttpDriver.doGet(HttpDriver.java:259)
  com.edu.scene.test.AddressList_Test.getAddressList_test(AddressList_Test.java:20)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:564)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
  org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
  java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
  java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
  java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
  java.base/java.lang.Thread.run(Thread.java:844)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
